src/lib/icon/icon.md

```diff
@@ -80,9 +80,32 @@ match the current theme's colors using the `color` attribute. This can be change
 
 ### Accessibility
 
-If an `aria-label` attribute is set on the `md-icon` element, its value will be used as-is. If not,
-the md-icon component will attempt to set the aria-label value from one of these sources:
-* The `alt` attribute
-* The `fontIcon` input
-* The name of the icon from the `svgIcon` input (not including any namespace)
-* The text content of the component (for ligature icons)
+Similar to an `<img>` element, an icon alone does not convey any useful information for a
+screen-reader user. The user of `<md-icon>` must provide additional information pertaining to how
+the icon is used.
+
+In thinking about accessibility, it is useful to place icon use into one of three categories:
+1. **Decorative**: the icon conveys no real semantic meaning and is purely cosmetic.
```

Comment:
I'm hesitant to always add `aria-hidden`, because the failure mode is that something that _should_ be visible to screen reader users isn't